src/zkevm_specs/evm/execution/block_coinbase.py

```diff
@@ -0,0 +1,20 @@
+from ..instruction import Instruction, Transition
```

Comment:
Just use `coinbase` as the file name?

Reply:
for `coinbase` case is ok, but I consider there are others block's op code (time, number etc), if use file name like  time.py, number.py , easily  cause ambiguous meaning compare to system files. 
So I prefer to keep prefix for all , like block_coinbase.py, block_time.py, block_number.py etc.